api/src/components/coauthor/controller.ts

```diff
@@ -253,13 +253,19 @@ export const updateConfirmation = async (
             });
         }
 
-        // Is the publication in draft?
+        // Is the publication in locked mode?
```

Comment:
here, we just need to check if currentStatus !== 'LOCKED', therefore confirmation cannot be  processed